src/main/resources/lib/advisor/validateOnLoad.jelly

```diff
@@ -18,21 +18,22 @@
   <div>
     <j:if test="${email!=null}">
       <j:set var="validationStatus" value="${app.getDescriptorByName('com.cloudbees.jenkins.plugins.advisor.AdvisorGlobalConfiguration').connectionTest(email)}"/>
-        <j:if test="${validationStatus.contains('connected')}">
+        <j:if test="${validationStatus.contains('service-operational')}">
           <div class="alert alert-success" role="alert">
-            <h4 class="alert-heading">${validationStatus}</h4>
+            <strong>Jenkins Healthy Advisor by CloudBees has been successfully connected.</strong>
           </div>
         </j:if>
-        <j:if test="${validationStatus.contains('xception')}">
+        <j:if test="${validationStatus.contains('tos-not-accepted')}">
           <div class="alert alert-warning" role="alert">
-            <h4 class="alert-heading">There was a connection failure to Jenkins Health Advisor by CloudBees. Check Advisor Server status at
-              <a href="https://www.cloudbeesstatus.com/">cloudbeesstatus.com</a>.</h4>
-            <p>${validationStatus}</p>
+            <strong>Terms and Conditions not accepted.</strong>
+            <p>To use Jenkins Health Advisor, please agree to <a href='https://www.cloudbees.com/jenkins/cloudbees-jenkins-advisor/terms-of-service'>our Terms and Conditions</a>.</p>
           </div>
         </j:if>
-        <j:if test="${validationStatus.contains('Terms of Service not accepted')}">
+        <j:if test="${validationStatus.contains('xception')}">
           <div class="alert alert-warning" role="alert">
-            <h4 class="alert-heading">${validationStatus}</h4>
+            <strong>Connection failure to Jenkins Health Advisor by CloudBees.</strong>
```

Comment:
Aside: what should we say to do when a user hits these issues?  Some would probably be easy to solve on their own, but should we provide a link out to documentation or say something like "Visit the CloudBees support site for information"

Reply:
You're right @kwhetstone Let's add a link to the troubleshooting KB https://support.cloudbees.com/hc/en-us/articles/115001213031
We are working in parallel to improve the documentation thus we might improve more all of this in the future